examples/k8s_cloud_deploy/README.md

```diff
@@ -0,0 +1,93 @@
+# Deploying a Kubernetes cluster on the cloud in 1-click with SkyPilot
+
+This example demonstrates how to deploy a Kubernetes cluster on the cloud with SkyPilot. For the purposes of this guide, we will use lambda cloud as the cloud provider, but you can change cloud providers by editing `cloud_k8s.yaml`.
+
```

Comment:
We should update this readme to point to lambda's official guide as well once that is published.